_data/authors.yml

```
@@ -90,3 +90,6 @@ Lucas Rademaker & Nefta Kanilmaz:
 
 Andreas Merkel, Kshitij Mathur:
     name: "Andreas Merkel and Kshitij Mathur"
+
+Marco Neumann & Usha Nemani:
+    name: "Marco Neumann & Usha Nemani"
```

Author change (as suggested below) and linking. You might wanna add you GitHub or twitter or whatever as well:

```suggestion
    name: "Usha Nemani & Marco Neumann"
    links:
      - label: "Marcos' Github"
        icon: "fab fa-fw fa-github-square"
        url: "https://github.com/crepererum"
```